R/zzz.R

```diff
@@ -0,0 +1,20 @@
+.onLoad <- function(libname, pkgname) {
```

Comment:
What is this for ? That function is not used anywhere else no ?

Reply:
the .onLoad()` function is a special one which is called when a package's namespace is loaded. This is a requirement/definition of the *alabaster* framework, that packages providing methods to import/export data need to *register* them in the package's `onLoad()` (which is usually provided in a source file called *zzz.R*).